backend/codesirius/problems/views/tag.py

```
@@ -51,48 +51,63 @@ class TagRetrieveUpdateDestroyAPIView(generics.RetrieveUpdateDestroyAPIView):
     def destroy(self, request, *args, **kwargs):
         pk = kwargs.get("pk")
         logger.info(f"Deleting tag with ID: {pk}")
-        super().destroy(request, *args, **kwargs)
-        logger.info(f"Tag with ID: {pk} deleted successfully")
-        return CodesiriusAPIResponse(
-            status_code=status.HTTP_204_NO_CONTENT,
-            message="Tag deleted",
-        )
+        try:
+            super().destroy(request, *args, **kwargs)
+            logger.info(f"Tag with ID: {pk} deleted successfully")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)